vic/drivers/image/src/vic_force.c

```diff
@@ -50,7 +50,7 @@ vic_force(void)
     extern parameters_struct   param;
 
     double                     t_offset;
-    float                     *fvar = NULL;
+    double                    *fvar = NULL;
```

Comment:
Confusing - I think `fvar` was specifically picked as a name to indicate a `float`. If you change it to double, change everything to `dvar`. If you want to change it to type-independent use `var`.